### ISSUE TITLE
Toolbar not loading since 1.161.3 with nextjs rewrites

### ISSUE THREAD

**Issue by FabianRueckert** [PostHog/posthog-js#1458]
Hi,

I am using nextjs rewrites [according to the docs](https://posthog.com/docs/advanced/proxy/nextjs)

```
  async rewrites() {
    return [
      {
        source: "/ingest/static/:path*",
        destination: "https://eu-assets.i.posthog.com/static/:path*",
      },
      {
        source: "/ingest/:path*",
        destination: "https://eu.i.posthog.com/:path*",
      },
      {
        source: "/ingest/decide",
        destination: "https://eu.i.posthog.com/decide",
      },
    ];
  },
```

Until release `1.161.2` the toolbar works fine, but since release `1.161.3`, the toolbar doesn't load correctly anymore, apparently due to a failed network call.

In `1.161.2` I can see in the chrome network tab, that the request to fetch the toolbar.js is made to

```
http://localhost:3000/ingest/static/toolbar.js?t=1728405000000
```

In `1.161.3` however,  when I look in the network tab, i can see a request to the url
```
http://localhost:3000/ingest/static/toolbar.js?v=1.167.0?&=1728404700000
```

 is made and returns with a 500 internal server error. Note that the `t` is missing before the `=1728404700000`.
If I re-add the `t` manually or if I disable the nextjs rewrite I correctly receive the JS for the toolbar.

**Comment by lucasra1** [comment]
Already have an PR ready that should fix that, we have the same problem.

https://github.com/PostHog/posthog-js/pull/1456

**Comment by lucasra1** [comment]
@pauldambra this can be closed. is fixed in [1.176.1](https://github.com/PostHog/posthog-js/releases/tag/v1.176.1)
